ui-manchette/src/components/helpers.ts

```diff
@@ -73,6 +82,29 @@ export const operationalPointsHeight = (
     }
   });
 
+export const computeTimeWindow = (trains: ProjectPathTrainResult[]) => {
+  const { minTime, maxTime } = trains.reduce(
+    (times, train) => {
+      if (train.space_time_curves.length === 0) return times;
+
+      const lastCurve = train.space_time_curves[train.space_time_curves.length - 1];
+      if (lastCurve.times.length === 0) return times;
+
+      const firstPoint = +new Date(train.departure_time);
```

Comment:
If I remember correctly, elsewhere in the app you had a comment between using `+` or `number` and it had been decided `Number` was better ?

```suggestion
      const firstPoint = Number(new Date(train.departure_time));
```